### ISSUE TITLE
Deploy postgresq/postgis replicated services with Docker

### ISSUE THREAD

**Issue by alexander-petkov** [alexander-petkov/wfas#8]
Add to the geoserver stack, or create a new one, hmm...

I am liking the idea of these different services decoupled. If one of the stacks is brought down, the other continues to function. This is on the expense of 2 separate deployments. 

Here is a good guide to get started:

https://info.crunchydata.com/blog/an-easy-recipe-for-creating-a-postgresql-cluster-with-docker-swarm

This task is tied with storing multidimensional coverages, and producing Time series... Rather than mucking with shapefiles, might as well bite the bullet and do it.

**Comment by alexander-petkov** [comment]
I will probably use the kartoza/postgis image

https://hub.docker.com/r/kartoza/postgis

**Comment by wmjolly** [comment]
I agree with you on this one completely. Decoupling things would make it more robust.

**Comment by alexander-petkov** [comment]
Figure out why postgres backup is not working.

**Comment by alexander-petkov** [comment]
> Figure out why postgres backup is not working.

Backups are working--enable by putting a new line in the `/etc/cron.d/backups-cron` file. 
Also, adjust env variables in `pgenv.sh` as needed. Might be better building a custom image with our settings.
